terraform/cloudwatch.tf.json.template.py

```diff
@@ -20,6 +20,20 @@
                         "unit": "Count"
                     }
                 }
+            },
+            "aws_cloudwatch_metric_alarm": {
+                f"{lambda_}_5xx": {
+                    "alarm_name": config.qualified_resource_name(lambda_ + '_5xx'),
+                    "comparison_operator": "GreaterThanOrEqualToThreshold",
+                    "evaluation_periods": 1,
+                    "period": 60 * 60,
```

Comment:
```suggestion
                    "period": 10 * 60,
```

But I think we might want to be able to configure all three parameters (# of periods, period and threshold).

We could do colons but I think `6x600s>10` with `x` and  `s>` being static separators instead of colons also serves as a mnemonic device. I would probably try parsing with a regex but string splitting/partitioning would work, too.